smarts/core/vehicle.py

```diff
@@ -631,11 +633,13 @@ def teardown(self, exclude_chassis=False):
         for sensor in [
             sensor
             for sensor in [
+                # pytype: disable=attribute-error
                 self._drivable_area_grid_map_sensor,
                 self._ogm_sensor,
                 self._rgb_sensor,
                 self._lidar_sensor,
                 self._driven_path_sensor,
+                # pytype: enable=attribute-error
```

Comment:
I think for these we should deal with the dynamic attributes this way:  https://github.com/google/pytype/blob/main/docs/faq.md#im-dynamically-populating-a-class--module-using-setattr-or-by-modifying-locals--globals-now-pytype-complains-about-missing-attributes-or-module-members-how-do-i-fix-this